UI/Source/CollectionViews/CollectionViewModelDataSource.swift

```diff
@@ -1057,10 +1062,15 @@ extension CollectionViewModelDataSource: NSCollectionViewDataSource {
 
     private func rebindViewAtIndexPath(_ indexPath: IndexPath, toViewModel viewModel: ViewModel) {
         guard let collectionView = collectionView else { return }
-        guard let item = collectionView.item(at: indexPath as IndexPath) as? CollectionViewHostItem else { return }
+        guard let item = collectionView.item(at: indexPath) as? CollectionViewHostItem else { return }
 
         willRebindViewModel(viewModel)
         item.hostedView?.bindToViewModel(viewModel)
+
+        // When rebinding, re-apply attributes so that the updated ViewModel is notified.
```

Comment:
Both of our CV host views pass layout attributes through to the underlying `View`. E.g.
https://github.com/dropbox/pilot/blob/master/UI/Source/CollectionViews/mac/CollectionViewHostItem.swift#L67

For clients that use custom layout attributes, doing so allows the `View` to receive them and react if desired.